webpack.config.js

```diff
@@ -19,7 +19,18 @@ module.exports = {
           'css-loader',
           'sass-loader'
         ]
-      }
+      },
+      {
+        test: /\.css$/,
+        loader: 'style-loader!css-loader'
+      },
+      // {
+      //   test: /\.(png|jpg|gif|svg|eot|ttf|woff|woff2)$/,
+      //   loader: 'url-loader',
+      //   options: {
+      //     limit: 10000
+      //   }
+      // }
```

Comment:
Do we need this section for this commit/merge? jw If not, maybe we remove for now so it's not moved into master